### PR TITLE
Carnival Theme: Line and Icon Spacing Fixes

### PR DESCRIPTION
**Pull request by clee1060**
https://github.com/openstax/cnx-recipes/issues/1221 <--- Line Spacing
https://github.com/openstax/cnx-recipes/issues/1224 <--- Icon Fix

Also it somehow fixed https://github.com/openstax/cnx-recipes/issues/1220? This is the mysterious hyperlink

### CODE REVIEW

**Comment by clee1060** [comment]
Please make sure I did all the OPTIONAL and REQUIRED stuff correctly I feel lik i probably forgot somewhere

File: styles/designs/carnival/parts/_linktolearning-components.scss [hunk 1]
```diff
@@ -39,15 +39,15 @@ $LinkToLearningBodyComponent: (
...
   )
 );
 
 $LinkToLearningTextComponent: (
   _name: "Text",
   _subselector: " > p",
   _properties: (
-    margin-top: enum('ValueSet:::OPTIONAL'),
-    margin-bottom: enum('ValueSet:::OPTIONAL'),
+    margin-top: enum('ValueSet:::NULLABLE'),
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_linktolearning-components.scss]: Should this be OPTIONAL?

Comment (tomjw64) [reply styles/designs/carnival/parts/_linktolearning-components.scss]: Additionally, post terminology change, I'm surprised that the framework doesn't error on this.

**Comment by helenemccarron** [comment]
Do not forget to run `./script/run ./script/build-styles` to generate all the files with your changes

**Pull request merged**